modules/admin_manual/pages/configuration/server/occ_command/file_commands.adoc

```diff
@@ -20,18 +20,17 @@ NOTE: These commands are not available in xref:maintenance-commands[single-user
 
 == The files:check-cache command
 
-The main purpose for this command is to clean the cache for objectstores (objectstore and files_primary_S3 apps) as
-primary storage, but it is not limited to only those. It can be used for any other storage type as long it is the primary storage.
+The main purpose for this command is to clean the cache for objectstores (objectstore and files_primary_S3 apps) as primary storage, but it is not limited to only those. 
```

Comment:
This sentence reads now different than before.
The point is, that it is _only_ usable on primary storage but not for classical mounts.

Reply:
The next sentence has that information, IMO ?

Reply:
You are right 🤦‍♂